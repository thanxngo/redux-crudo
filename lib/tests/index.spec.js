"use strict";

var _index = require("../index");

var _index2 = _interopRequireDefault(_index);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe("Index", function () {
    it("should export package", function () {
        expect(JSON.stringify(_index2.default)).toMatchSnapshot();
    });
}); /* eslint-env jest */