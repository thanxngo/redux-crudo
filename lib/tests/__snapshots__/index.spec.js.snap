// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should export package 1`] = `undefined`;
